readme.md

```diff
@@ -18,15 +18,27 @@ $ npm install --global osx-wallpaper
 [Download the binary](https://github.com/sindresorhus/osx-wallpaper/releases/latest) and put it in `/usr/local/bin`.
 
 
-## Usage
+## Binary usage
 
 ```sh
-# set
+# Set wallpaper on main screen
 wallpaper unicorn.jpg
 
-# get
+# List connected screens (screen ID:screen name)
+wallpaper --screeninfo
+> 69732482:Color LCD
+> 188845747:SM2333T
+
+# Set wallpaper on specific screen (replace screenid with your screen ID)
+wallpaper screenid unicorn2.jpg
```

Comment:
Left to right.

Maybe we should consider only setting the main screen by default (when no argument), to be consistent with the getter. And add a `--all` flag for setting to all screens?